test/fixtures/lightscript/if-expression/inner-if-stmt-return/options.json

```diff
@@ -0,0 +1,3 @@
+{
+  "throws": "'return' outside of function (4:6)"
+}
```

Comment:
(Sorry if this is a duplicate, I tried to leave this comment and it somehow got sucked into the pattern matching discussion.)

I think people are going to be extremely confused by this error message, since the `return` is definitely not outside of a function. It should probably say something like: `'return' is not allowed in expressions.`

This may be an upstream PR to babylon as well, since a `doExpression` should give that message as well.